src/containers/PerfherderGraphContainer/index.jsx

```diff
@@ -65,7 +65,7 @@ PerfherderGraphContainer.propTypes = {
       seriesConfig: PropTypes.shape({
         extraOptions: PropTypes.arrayOf(PropTypes.string),
         frameworkId: PropTypes.number.isRequired,
-        option: PropTypes.string.isRequired,
+        option: PropTypes.string,
```

Comment:
For none required properties, define a default prop. Example:

```jsx
PerfherderGraphContainer.defaultProps = {
  option: null,
  // ...
}
```

Reply:
This is more involved

Reply:
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/312